templates/about/publish.html

```diff
@@ -7,5 +7,99 @@
 {% block meta_title %}About Publishing | Snapcraft{% endblock %}
 
 {% block about_content %}
+  <div class="p-strip is-shallow u-no-padding--top">
+    <h1 class="p-heading--2">Snapcraft lets you build with the tools you already use</h1>
+    <p>
+      With support for many languages out of the box, getting started with Snapcraft can be as simple as <a href="/docs/snapcraft-overview">installing and running snapcraft</a>.
+    </p>
+    {{
+      image(
+        url="https://assets.ubuntu.com/v1/143fff51-DO+NOT+USE%21.jpg",
```

Comment:
Do you know why this file is named "DO NOT USE"?

Reply:
Not really, but I assume they were trying to make it obvious not to use this asset. I use it as a placeholder image until Lily is back and gives me the final images, so I don't think it's a problem.